tests/acceptance/provideSearchHeadless.ts

```diff
@@ -1,4 +1,4 @@
-import { provideAnswersHeadless } from '../../src';
+import { provideHeadless } from '../../src';
```

Comment:
Could we rename this file to provideHeadless.ts instead of provideSearchHeadless.ts?